packages/miniflare/src/index.ts

```diff
@@ -1144,6 +1145,14 @@ export class Miniflare {
 			const workerName = workerOpts.core.name ?? "";
 			const isModulesWorker = Boolean(workerOpts.core.modules);
 
+			if (workerOpts.workflows.workflows) {
+				for (const workflow of Object.values(workerOpts.workflows.workflows)) {
+					// This will be the UserWorker, or the vitest pool worker wrapping the UserWorker
+					// The workflows plugin needs this so that it can set the binding between the Engine and the UserWorker
+					workflow.scriptName ??= this.#workerOpts[0].core.name;
```

Comment:
```suggestion
					workflow.scriptName ??=  workerOpts.core.name;
```
or is this just assuming there will only be one user worker, which is the first worker...?